src/Data/Traversable.purs

```diff
@@ -60,6 +62,39 @@ sequenceDefault :: forall t a m. (Traversable t, Applicative m) =>
 sequenceDefault tma = traverse id tma
 
 
+newtype Id a = Id a
+
+runId :: forall a. Id a -> a
+runId (Id a) = a
```

Comment:
Actually, the whole data type isn't even needed.

Since this library depends on `purescript-prelude` and `(->) r` has an `Applicative` instance, you should be able to say:

```purescript
mapDefault :: forall b a t. (Traversable t) => (a -> b) -> t a -> t b
mapDefault f m = traverse (const <<< f) m $ unit
```